### PR TITLE
fix(DB/Nagrand): add missing ability to Ravenous Windroc

### PR DESCRIPTION
**Pull request by elthehablo**
## Changes Proposed:
-  change comment on Windroc to be more descriptive
-  add ability

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/16192

## SOURCE:

see issue

## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/ravenous_windroc.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+--
+UPDATE `smart_scripts` SET `comment` = 'Windroc - In Combat - Cast Eagle Claw' WHERE `entryorguid` = 17128 AND `source_type` = 0 AND `id` = 0;
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `Entry` = 18220;
+
+DELETE FROM `smart_scripts` WHERE `entryorguid` = 18220 AND `source_type` = 0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(18220, 0, 0, 0, 0, 0, 100, 0, 2500, 5000, 20000, 25000, 0, 11, 30285, 32, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Ravenous Windroc - In Combat - Cast Eagle Claw');
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/ravenous_windroc.sql:7]: Where did you get these timers?
```suggestion
(18220, 0, 0, 0, 0, 0, 100, 0, 2500, 5000, 20000, 25000, 0, 11, 30285, 32, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Ravenous Windroc - In Combat - Cast \'Eagle Claw\'');
```

File: data/sql/updates/pending_db_world/ravenous_windroc.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+--
+UPDATE `smart_scripts` SET `comment` = 'Windroc - In Combat - Cast Eagle Claw' WHERE `entryorguid` = 17128 AND `source_type` = 0 AND `id` = 0;
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/ravenous_windroc.sql:2]:
```suggestion
UPDATE `smart_scripts` SET `comment` = 'Windroc - In Combat - Cast \'Eagle Claw\'' WHERE `entryorguid` = 17128 AND `source_type` = 0 AND `id` = 0;
```

File: data/sql/updates/pending_db_world/ravenous_windroc.sql (hunk 1 above)
Comment (elthehablo) [reply data/sql/updates/pending_db_world/ravenous_windroc.sql:7]: timers copy pasted from Windroc

Comment (Gultask) [reply data/sql/updates/pending_db_world/ravenous_windroc.sql:7]: Cmangos is more accurate than old Sunwell, so I added it in the suggestion 😛

**Review by Nyeriah — APPROVED** [review]

**Pull request merged**